ohsomeTools/gui/ohsome_spec.py

```diff
@@ -243,60 +223,24 @@ def __prepare_ohsome_time_parameter(
             dates = f"{date_start},{date_end}"
         return dates
 
-    def __get_selected_polygon_layers_geometries(self) -> []:
-        layer_list = []
-        polygon_layer_list = self.dlg.layer_list
-        for idx in range(polygon_layer_list.count()):
-            item: str = polygon_layer_list.item(idx).text()
-            layers = QgsProject.instance().mapLayersByName(item)
-            layers = [
-                layer
-                for layer in layers
-                if layer.geometryType() == QgsWkbTypes.PolygonGeometry
-            ]
-            if len(layers) > 1:
-                raise exceptions.TooManyInputsFound(
-                    str("error"),
-                    # error,
-                    "Found too many input layers with the same name. Use unique names for your layers.",
-                )
-            layer_list.extend(layers)
-        geojsons = [
-            QgsJsonExporter(lyr)
-            .exportFeatures(lyr.getFeatures())
+    def _get_selected_polygon_layers_geometries(self) -> []:
+        layer = self.dlg.layer_input.currentLayer()
```

Comment:
Why are we switching from a list of layers to a single layer here?

Reply:
As discussed in #60 

> The option to add multiple point or polygon layers seems a bit unnecessary and can be dropped

So we no longer need a list to handle multiple layers.